client/assets/scss/colors.scss

```diff
@@ -1,4 +1,59 @@
-$blue: rgba(52, 88, 170, 1);
-$grey: rgba(106, 106, 106, 1);
-$red: rgba(204, 0, 0, 1);
-$white: rgba(255, 255, 255, 1);
\ No newline at end of file
+//blue
+$blue-whale: rgba(15, 47, 63);
+$blue-whale-75: rgba(15, 47, 63, .75);
```

Comment:
Let's not add the ones we aren't using

Reply:
i can comment them out and comment them back in when in use or as a note give it a rule to keep it at 75, 50 and 25. no weird ones like .52 opacity